_posts/2021-2-11-Overview-of-Today's-Work.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Overview of Today's Work
+description: 
+summary: 
+tags: 
+---
+Our starting task for today was to continue progressing our typing accuracy and speed on Typing club. Once we finished, Kajari gave us a link to a website to practice and introduce us to code we will use to create a small video game. [Here is the link tot eh webstie used for the practice](https://developer.mozilla.org/en-US/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript)We first created a new folder and added the html elements onto the folder named breakout. The practice consisted of 10 chapters, where each chapter focused on creating certain elements and functions for the small ball point game. After completing the practice, we made a new repository named after the file, and pushed up the repository through terminal to pop up on our github.io's. [Here is the link for the website we used to push the repository](https://docs.github.com/en/github/importing-your-projects-to-github/adding-an-existing-project-to-github-using-the-command-line) After pushing up the repository, we invited Kajari as a collaborator and sent her the link to our completed practice game. [Here is the completed game](https://osvaldo178.github.io/breakout/).
```

Comment:
correct spelling: ` tot eh webstie `

Reply:
`JavaScript)We` : Put a period after ending the last sentence and space before starting the new one.

Reply:
Start a new paragraph for: `After completing the practice,`

Reply:
Start a new paragraph after: `speed on Typing club.`